beard/metrics/string_distance.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you rename this file to simply `string.py`? The `_distance` suffix is redundant with the fact that the file is already within a folder named "metrics".